tests/dummy/app/controllers/modules/modal.js

```diff
@@ -1,5 +1,6 @@
 /* global alert */
 import Ember from 'ember';
+import $ from 'jquery';
```

Comment:
because I received notice the error of `no-unused-vars`

https://github.com/aureliosaraiva/Semantic-UI-Ember/blob/33b7d17225c580364178ce13a2cbf23b1aae550e/tests/dummy/app/controllers/modules/modal.js#L8